extras/uiuc-2016-04/README.md

```diff
@@ -0,0 +1,5 @@
+# Sample Graders #
+
+Brennan Saeta <@saeta> authored these graders as part of a grader-authoring workshop conducted at
+UIUC in April 2016. See [the slides](https://docs.google.com/presentation/d/1qRU7HM5JBFH3wtTW01GRZrs5X-EtL_HRjY-aDtqqiSc/present)
+for further background.
```

Comment:
Could we add more information about what this grader is like high level stuff and to what audience this might be useful to?